src/System.Globalization.Calendars/tests/CalendarHelpers.cs

```diff
@@ -0,0 +1,189 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Globalization.Tests
+{
+    public static class CalendarHelpers
+    {
+        private static int MaxEra(Calendar calendar) => calendar.GetEra(calendar.MaxSupportedDateTime);
+        private static int MinEra(Calendar calendar) => calendar.GetEra(calendar.MinSupportedDateTime);
+
+        private static int YearOffset(Calendar calendar)
+        {
+            if (calendar is KoreanCalendar)
```

Comment:
It is better to calculate this instead of hardcoding the value. something like Calendar.GetYear(Calendar.MinSupportedDateTime) -1